index.js

```diff
@@ -33,6 +33,16 @@ const OptionsType = new Archetype({
     $type: 'string',
     $required: false,
     $default: null
+  },
+  expiresColumn: {
+    $type: 'string',
+    $required: true,
+    $default: 'expires'
+  },
+  expiresAfterSeconds: {
```

Comment:
Why not reuse `expires`?

Reply:
Because the existing behaviour for Mongo is for that to be set at `0` and in Cosmos Mongo is needs to be set to the expiry in seconds (eg, 14 days). If you're happy with changing the existing behaviour (which I think would make the sessions in Mongo remain in the DB for twice the length as desired) then I can re-use that, but that seems a change we don't want to make to me